core/src/main/java/org/infinispan/configuration/parsing/Parser.java

```diff
@@ -2380,16 +2233,18 @@ private void parseClusterLoader(XMLExtendedStreamReader reader, ConfigurationBui
 
    protected void parseFileStore(XMLExtendedStreamReader reader, ConfigurationBuilderHolder holder) throws XMLStreamException {
       SingleFileStoreConfigurationBuilder storeBuilder = holder.getCurrentConfigurationBuilder().persistence().addSingleFileStore();
+      String path = null;
+      String relativeTo = null;
       for (int i = 0; i < reader.getAttributeCount(); i++) {
          String value = reader.getAttributeValue(i);
          Attribute attribute = Attribute.forName(reader.getAttributeLocalName(i));
          switch (attribute) {
             case RELATIVE_TO: {
-               log.ignoreXmlAttribute(attribute);
+               relativeTo = (String) reader.getProperty(value);
```

Comment:
Nice addition. Without this I was having to do things like [this](https://github.com/jboss-container-images/jboss-datagrid-7-openshift-image/blob/datagrid73-dev/services/functional-tests/src/test/java/org/infinispan/online/service/datagrid/PersistedIndexSurvivesTest.java#L198).